src/adapters/arpalazio.js

```diff
@@ -141,11 +141,12 @@ export const getStream = function (cityName, url, averagingPeriod, source, orgUr
     .map(
       ([date1, date2, ...x]) => {
         const timestamp = (averagingPeriod.value === 1)
-          ? moment.tz(`${year} ${date1} ${date2}`, 'YYYY DDD HH', timezone)
-          : moment.tz(`${year} ${date2}`, 'YYYY DDD', timezone);
+          ? DateTime.fromObject({year, ordinal: date1, hour: date2}).setZone(timezone)
+          : DateTime.fromObject({year, ordinal: date2}).setZone(timezone);
+
         const date = {
-          utc: timestamp.toDate(),
-          local: timestamp.format()
+          utc: timestamp.toUTC().toFormat("yyyy-MM-dd'T'HH:mm:ss'Z'"),
+          local: timestamp.toFormat("yyyy-MM-dd'T'HH:mm:ssZZ")
```

Comment:
Why is Z quoted in the line above but not here?

Reply:
I have found it useful to explicitly set the format, I am open to suggestions.